admin/load.php

```diff
@@ -540,21 +543,29 @@ function perflab_enqueue_modules_page_scripts() {
  */
 function perflab_activate_plugin() {
 	// Do not proceed if plugin query arg is not present.
-	if ( empty( $_GET['plugin'] ) ) {
+	if ( empty( $_GET['plugin_slug'] ) ) {
```

Comment:
Why change the field name? I believe this will break the other plugin activation functionality that already existed via #651.

Let's implement this to be compatible with both ways:
* We should keep using the existing `plugin` name, to not have to update unrelated code.
* We should check whether `plugin` is a plugin slug or a plugin basename and handle accordingly:
    * If it's a basename, everything should work as it did before.
    * If it's a plugin slug (and _only_ then), we should run the logic to find the plugin basename (i.e. the `get_plugins()` call etc. you already have).